homework-submission/react-hyf/react-hyf-client/src/components/navbar/navbar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const NavBar = () => {
+  return (
+    <div className="menu">
+      <ul class="">
```

Comment:
A super easy fix:)

<img width="450" alt="Screen Shot 2019-07-17 at 06 51 51" src="https://user-images.githubusercontent.com/25272822/61348126-ce6b2a80-a85f-11e9-9e3d-2ccb569d635f.png">

Reply:
You are right , it is className 😅